### ISSUE TITLE
Training scheme of the model

### ISSUE THREAD

**Issue by kevinisbest** [ox-vgg/vgg_face2#25]
Hi, I have some questions about the training setting.

In [VGGFace](http://cis.csuohio.edu/~sschung/CIS660/DeepFaceRecognition_parkhi15.pdf), the author initially train a VGG16 based N-ways classifier and then remove the FC layer to fine-tune the network with triplet-loss. 

But in VGGFace2, did you just use softmax loss to generate the ResNet-50 pre-trained model  on MS-Celeb-1M and also apply softmax loss to fine-tune on VGGFace2 dataset?
Thank you!

**Comment by WeidiXie** [comment]
yes, remove the pertained classifier from MS-Celeb-1M (lots of classes), and fine-tune on VGGFace2 with another classifier (around 8641 classes).

**Comment by kevinisbest** [comment]
Thanks for rapid answer, so the reason not use triplet-loss is the convergence time or something?

And another question, have you ever tried the SOTA loss functions like Center loss, L-Softmax, A-Softmax Loss or AM-Softmax?

**Comment by WeidiXie** [comment]
nope, the paper aims to propose a very baseline for the face recognition research, so we didn't try any sophisticated loss or architectures.

**Comment by kevinisbest** [comment]
OK! I got it!
Thank you again!
